src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -57,11 +57,114 @@ void createStation() {
      */
     // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
 
+    @Test
+    @DisplayName("지하철역 목록 조회")
+    void getStationList(){
```

Comment:
코드 포맷팅 부분 확인해주시면 좋을 것 같습니다.
```suggestion
    void getStationList() {
```